### PR TITLE
[HTTP-cache] Added SSI

### PR DESCRIPTION
**Pull request by Ioni14**
Hello!

This is my first "big" contribution to Symfony docs. :)
It has many similarities with the ESI one but I think it can be better known and used with a specific doc.

I think it's Symfony 2.6 (!) when the SSI feature has been merged, I hope I'm right.

I guess there are many typos, I'll fix them as quickly as possible.

Thanks for reviewing. ;)

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@Ioni14 thanks for this contribution. I'll try to review it soon.

You are right that this was introduced in Symfony 2.6 ... and it's part of #2129, a pending issue since Symfony 2.2 !!

File: http_cache/ssi.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    {# ... or a URL #}
+    {{ render_ssi(url('profile_gdpr')) }}
+
+The ``render_ssi`` twig helper will generate something like:
+
+.. code-block:: html
+
+    <!--#include virtual="/_fragment?_hash=abcdef1234&_path=_controller=App\Controller\ProfileController::gdpr" -->
+
+``render_esi`` ensures that SSI directive are generated only if the request
```
Comment (ternel) [inline http_cache/ssi.rst]:
```suggestion
``render_ssi`` ensures that SSI directive are generated only if the request
```

File: http_cache/ssi.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+Working with Server Side Includes
+=================================
+
+In a similar way as :doc:`ESI (Edge Side Includes) <esi>`, SSI can be used to
+control HTTP caching on fragments of a response. The most important
+difference that is SSI is known directly by most web servers like
+`Apache <https://httpd.apache.org/docs/current/en/howto/ssi.html>`_,
+`Nginx <https://nginx.org/en/docs/http/ngx_http_ssi_module.html>`_ etc.
+
+The SSI instructions are done in HTML comments:
```
Comment (OskarStark) [inline http_cache/ssi.rst]: ... done via...

File: http_cache/ssi.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            <!-- ... some content -->
+
+            <!-- Embed the content of another page here -->
+            <!--#include virtual="http://..." -->
+
+            <!-- ... more content -->
+        </body>
+    </html>
+
+There is some other `available directives
```
Comment (OskarStark) [inline http_cache/ssi.rst]: There are...

**Comment by OskarStark** [comment]
Thank you Thomas.

**Pull request merged**